README.rst

```diff
@@ -332,31 +330,25 @@ Appended in this repository are three more docker-compose files for different ki
 Test cornflow
 ~~~~~~~~~~~~~~~~~~
 
-Create a new test database::
-
-    sudo su - postgres
-    psql -c "create database cornflow_test"
-    exit
-
 Then you have to run the following commands::
 
-    export FLASK_APP=cornflow.app
     export FLASK_ENV=testing
 
 Finally you can run all the tests with the following command::
 
```

Comment:
Why do do you not want to run the test directly with coverage?

Reply:
good question. I've always seen the tests being run by `unittest` directly in other places. Coverage is to get the coverage, right? It seems it uses `unittest` under the hood so in the end is the same thing. When we need the coverage, then it makes sense to run `coverage` (as in the last command).